tests/factories.py

```diff
@@ -185,6 +185,11 @@ class Meta:
 class ProjectFactory(AbstractNodeFactory):
     category = 'project'
 
+class PublicFilesFactory(AbstractNodeFactory):
+    title = "Public Files"
```

Comment:
As per the [dev docs](http://cosdev.readthedocs.io/en/latest/style_guides/python.html#general-naming-guidelines), this should use single quotes.